guides/v2.1/cloud/reference/docker-config.md

```diff
@@ -18,7 +18,7 @@ The [Magento Cloud Docker repository](https://github.com/magento/magento-cloud-d
     -  PHP-FPM - version 7 and later
 -  NGINX:  magento/magento-cloud-docker-nginx
 
-The Cloud Tools provides a `docker:build` command to generate the Docker Compose configuration. Also, you can specify a version using one of the following options:
+The ece-tools provides a `docker:build` command to generate the Docker Compose configuration. Also, you can specify a version using one of the following options:
```

Comment:
> "The `ece-tools` package..."

The "the" makes this awkward and I understand why you would avoid starting the sentence with "ece-tools" because that would also be awkward.